Jenkinsfile

```diff
@@ -216,73 +221,6 @@ pipeline {
                         }
                     }
                 }
-                stage('experimental') {
-                    when {
-                        beforeAgent true
-                        expression { params.experimental }
-                    }
-                    agent { label 'amd64 && ubuntu-1804 && overlay2' }
-
-                    stages {
-                        stage("Print info") {
-                            steps {
-                                sh 'docker version'
-                                sh 'docker info'
-                                sh '''
-                                echo "check-config.sh version: ${CHECK_CONFIG_COMMIT}"
-                                curl -fsSL -o ${WORKSPACE}/check-config.sh "https://raw.githubusercontent.com/moby/moby/${CHECK_CONFIG_COMMIT}/contrib/check-config.sh" \
-                                && bash ${WORKSPACE}/check-config.sh || true
-                                '''
-                            }
-                        }
-                        stage("Build dev image") {
-                            steps {
-                                sh 'docker build --force-rm --build-arg APT_MIRROR -t docker:${GIT_COMMIT}-exp .'
-                            }
-                        }
-                        stage("Integration tests") {
-                            steps {
-                                sh '''
-                                docker run --rm -t --privileged \
-                                  -v "$WORKSPACE/bundles:/go/src/github.com/docker/docker/bundles" \
-                                  --name docker-pr$BUILD_NUMBER \
-                                  -e DOCKER_EXPERIMENTAL=y \
```

Comment:
ah yes; copied it from here

but it probably only looks for "any non empty value"?

Reply:
Yup; only looks for "non-empty"; https://github.com/moby/moby/blob/a6aa11a3ef17d30dc06d63b28d0aed4e92c96843/hack/make/.integration-daemon-start#L60-L64